packages/Webkul/Admin/src/Resources/views/catalog/categories/create.blade.php

```diff
@@ -102,6 +102,18 @@
                                 </span>
 
                                 <span class="control-info mt-10">{{ __('admin::app.catalog.categories.image-size') }}</span>   
+
+                                <label>{{ __('admin::app.catalog.categories.category_banner') }}</label>
+
+                                <image-wrapper button-label="{{ __('admin::app.catalog.products.add-image-btn-title') }}" input-name="category_banner" :multiple="false"></image-wrapper>
+
+                                <span class="control-error" v-if="{!! $errors->has('image.*') !!}">
+                                    @foreach ($errors->get('image.*') as $key => $message)
+                                        @php echo str_replace($key, 'Image', $message[0]); @endphp
+                                    @endforeach
+                                </span>
+
+                                <span class="control-info mt-10">{{ __('admin::app.catalog.categories.image-size') }}</span>   
```

Comment:
@Rachana1005 
Need to mention the correct suggestion size for the banner image. Here you used the suggestion of the Image size.